packages/hydrooj/src/model/problem.ts

```diff
@@ -431,11 +431,17 @@ export class ProblemModel {
         } else {
             throw new ValidationError('file', null, 'Invalid file');
         }
+        const summary = { readme: [], problems: [] };
         try {
             const problems = await fs.readdir(tmpdir, { withFileTypes: true });
             for (const p of problems) {
                 const i = p.name;
-                if (!p.isDirectory()) continue;
+                if (!p.isDirectory()) {
```

Comment:
It is usually meaningless for a problemset to contain multiple readme file. While it's useful for i18n use cases (README.zh.md, README.en.md), it might be better to select a file according to user's language

Reply:
Choose one of the readme file only.